po/zh_CN.po

```diff
@@ -69,51 +38,89 @@ msgstr ""
 " <longdesc lang=\"zh\">实例属性可用于所有stonith类资源，并由Pacemaker的fence"
 "守护程序使用（以前称为stonithd）</longdesc>\n"
 
-#: daemons/fenced/pacemaker-fenced.c:1484
+#: daemons/fenced/pacemaker-fenced.c:1467
 #, c-format
 msgid ""
 " <shortdesc lang=\"en\">Instance attributes available for all \"stonith\"-"
 "class resources</shortdesc>\n"
 msgstr ""
 " <shortdesc lang=\"zh\">可用于所有stonith类资源的实例属性</shortdesc>\n"
 
-#: daemons/fenced/pacemaker-fenced.c:1508
-#, c-format
+#: daemons/fenced/pacemaker-fenced.c:1490
+#, fuzzy, c-format
 msgid ""
 "    <longdesc lang=\"en\">Some devices do not support the standard 'port' "
-"parameter or may provide additional ones.\n"
-"Use this to specify an alternate, device-specific, parameter that should "
-"indicate the machine to be fenced.\n"
-"A value of 'none' can be used to tell the cluster not to supply any "
-"additional parameters.\n"
-"     </longdesc>\n"
+"parameter or may provide additional ones. Use this to specify an alternate, "
+"device-specific, parameter that should indicate the machine to be fenced. A "
+"value of '%s' can be used to tell the cluster not to supply any additional "
+"parameters.\n"
+"    </longdesc>\n"
 msgstr ""
-"    <longdesc lang=\"zh\">某些设备可能不支持使用标准的'port'(端口)参数，也可能会提供额外的端口参数。\n"
-"使用此参数可以为需要fence(防护)的机器指定一个备用的，专用于该设备的参数,该参数应指出要fence的机器。\n"
+"    <longdesc lang=\"zh\">某些设备可能不支持使用标准的'port'(端口)参数，也可"
+"能会提供额外的端口参数。\n"
+"使用此参数可以为需要fence(防护)的机器指定一个备用的，专用于该设备的参数,该参"
+"数应指出要fence的机器。\n"
 "使用值'none'可用来告诉集群不提供任何额外的参数\n"
```

Comment:
You'll need to replace 'none' with '%s' here.

Reply:
updated commit